shared/style/input_areas.css

```diff
@@ -211,7 +211,7 @@ input:invalid,
 textarea:invalid,
 .skin-dark input:invalid,
 .skin-dark textarea:invalid {
-  color: #b90000;
+  color: #000;
```

Comment:
are you sure that these (here and at line 229) colours shall be common for entire platform?